### ISSUE TITLE
coding standards: imported settings are not working

### ISSUE THREAD

**Issue by vitek-rostislav** [shopsys/shopsys#2012]
### What is happening
Some rules (or maybe all? I have randomly discovered `NoUnusedImportsFixer` and `NoWhitespaceInBlankLineFixer`) defined in `easy-coding-standard-yaml` in `shopsys/coding-standards` package are not applied in our project, even though we include these settings in our `easy-coding-standard-yaml`:
```
imports:
    - { resource: '%vendor_dir%/shopsys/coding-standards/easy-coding-standard.yml', ignore_errors: true  }
```

When I add the fixer in my config, everything works properly.

### Expected result
I do not have to re-define rules (checks/fixers) that are already defined in the coding-standards package, i.e. the imports work properly

**Comment by vitek-rostislav** [comment]
it seems like `%vendor_dir%` is not set properly :thinking:

**Comment by sslt** [comment]
FYI: Will be fixed within fortnight.

**Comment by s3tezsky** [comment]
Hi @vitek-rostislav thanks for reporting it!

It seems to be related to https://github.com/symplify/symplify/issues/1795

**Comment by s3tezsky** [comment]
fixed in #2026
